docs/dyndns/configure.rst

```diff
@@ -108,16 +109,21 @@ To install ddclient, run ``sudo apt-get install ddclient``. If a configuration
 dialog does not appear automatically, use ``sudo dpkg-reconfigure ddclient`` to
 start the configuration process.
 
-In the configuration process, select "other" dynamic DNS service provider, and
-enter ``update.dedyn.io`` as the dynamic DNS server. Next, tell ddclient to use
-the "dyndns2" protocol to perform updates. Afterwards, enter the username and
-the token secret that you received during registration. Last, tell ddclient how to
-detect your IP address, your domain name and the update interval.
+In the configuration process, select "other" dynamic DNS service provider. 
+Next, tell ddclient to use the "dyndns2" protocol to perform updates.
+Enter ``update.dedyn.io`` as the dynamic DNS service. Afterwards, enter the domain 
+you want to update as the username.
+As your password, use a token that you generated under ``https://desec.io/tokens``.
+Tell ddclient how to detect your IP address. You can use any webservice on the list
+or select ``others`` and then insert ``https://checkipv4.dedyn.io/`` to check your IPv4
+or ``https://checkipv6.dedyn.io/`` to check your IPv6.
+Insert the update interval and then the domain name to update.
 
 **Note:** As of the time of this writing, ddclient does not use an encrypted
 HTTPS connection by default. To enable it, open ``/etc/ddclient.conf`` and add
 ``ssl=yes`` above the ``server=`` statement. We **strongly recommend** doing
-so; otherwise, your credentials will be exposed during transmission.
+so; otherwise, your credentials will be exposed during transmission. Also webservices
+like ``https://checkipv4.dedyn.io/`` for IP checks depend on it!
```

Comment:
I'm not sure that checking with IP with a HTTPS URL breaks when you don't see `ssl=yes`. Isn't this setting for the update request?

Reply:
That's not true; I just tested it. Doing an update via HTTPS using IP discovery via HTTPS works, and the correct IP address is discovered and set, even when `ssl=yes` is not configured.

Reply:
for the update.dedyn.io, you don't need it. 
But if you set 
`use=web, web=checkipv4.dedyn.io/ \`
without https, then you need 
`ssl=yes`

If you set 
`use=web, web=https://checkipv4.dedyn.io/ \`
you don't need it.

I also just tested this on an Ubuntu test machine

Reply:
and of course I don't know how other IP checkers handle this. Maybe someone besides you also forces you to use https for an IP check.

Reply:
Strange. 

I tested this on a Ubuntu machine. Then I wanted to implement this on my production debian host. 
On Debian even setting ssl=yes seems to not be enough.
I have to set https://checkipv4.dedyn.io, otherwise it will fail.

Reply:
Makes sense, removed the line. 
Added the test line form manual config. 
Feel free to delete it if you don't like it.